db/wal_manager_test.cc

```diff
@@ -167,15 +168,16 @@ namespace {
 uint64_t GetLogDirSize(std::string dir_path, Env* env) {
   uint64_t dir_size = 0;
   std::vector<std::string> files;
-  env->GetChildren(dir_path, &files);
-  for (auto& f : files) {
-    uint64_t number;
-    FileType type;
-    if (ParseFileName(f, &number, &type) && type == kWalFile) {
-      std::string const file_path = dir_path + "/" + f;
-      uint64_t file_size;
-      env->GetFileSize(file_path, &file_size);
-      dir_size += file_size;
+  if (env->GetChildren(dir_path, &files).ok()) {
```

Comment:
As this is a test should we also have an EXPECT_OK on the status of `GetChildren` here?